apps/judicial-system/backend/src/app/formatters/pdfHelpers.ts

```diff
@@ -111,123 +113,147 @@ export const addPoliceStar = (doc: PDFKit.PDFDocument) => {
 
 export const addIndictmentConfirmation = (
   doc: PDFKit.PDFDocument,
-  confirmedBy: string,
-  institutionName: string,
-  createdDate: Date,
+  confirmation: IndictmentConfirmation,
```

Comment:
Note that indictmentPdf is importing from pdfHelpers and vice versa.

Please consider either of these two approaches:
- Move the definition of IndictmentConfirmation to pdfHelpers, or
- call addIndictmentConfirmation with individual arguments rather than a confirmation object.